lang/pt_BR.json

```
@@ -1,11 +1,169 @@
+    "Number Format": "Formato de Número",
+    "Overview": "Visão Geral",
+    "Page :page of :total": "Página :page de :total",
+    "Pink": "Rosa",
+    "Play": "Jogar",
+    "Privacy Policy": "Políticas de Privacidade",
+    "Privacy": "Privacidade",
+    "Rank": "Classificação",
+    "Red": "Vermelho",
+    "Register": "Registrar",
+    "Save Settings": "Salvar Configurações",
+    "Search": "Pesquisar",
+    "Sign In": "Entrar",
+    "Sign Out": "Sair",
+    "Signed in as :user": "Logado como :user",
+    "Site Preferences": "Preferências do Site",
+    "Site": "Site",
+    "Subject": "Assunto",
+    "System Default": "Padrão do Sistema",
+    "Theme": "Tema",
+    "Timezone": "Fuso Horário",
+    "Username": "Nome de Usuário",
+    "Users Code of Conduct": "Código de Conduta dos Usuários",
+    "account": "Conta",
+    "accounts": "Contas",
     "achievement": "Conquista",
     "achievements": "Conquistas",
-    "Language": "Língua",
-    "Timezone": "Fuso horário",
-    "Number Format": "Formato numérico",
-    "Date/Time Format": "Formato de data/hora",
-    "Site Preferences": "Preferências do site"
+    "achievements-progress": "Progresso das Conquistas",
+    "api": "API",
+    "application": "Aplicação",
+    "applications": "Aplicações",
+    "avatar": "Avatar",
+    "avatars": "Avatares",
+    "badge": "Medalha",
+    "badges": "Medalhas",
+    "by :user on :date": "de :user em :date",
+    "cancel": "Cancelar",
+    "comment": "Comentar",
+    "comments": "Comentários",
+    "connect": "Conecte",
+    "currently playing": "Jogando no momento",
+    "default": "Padrão",
+    "display-preferences": "Preferências de Tela",
+    "edit": "Editar",
+    "email-change": "Mudar E-Mail",
+    "emulator": "Emulador",
+    "emulators": "Emuladores",
+    "entries": "Entradas",
+    "entry": "Entrada",
+    "faq": "FAQ",
+    "feedback": "Feedback",
+    "forum": "Fórum",
+    "forums": "Fóruns",
+    "friend": "Amigo",
+    "friends": "Amigos",
+    "game": "Jogo",
+    "games": "Jogos",
+    "has signed in": "se registrou em",
+    "integration": "Integração",
+    "integrations": "Integrações",
+    "last seen": "Visto por último",
+    "latest release": "Último Lançamento",
+    "leaderboard": "Placar de Líder",
+    "leaderboards": "Placar de Líderes",
+    "legal": "Legal",
+    "libraries": "Bibliotecas",
+    "library": "Biblioteca",
+    "logo": "Logo",
+    "manage": "Gerenciar",
+    "message": "Mensagem",
+    "messages": "Mensagens",
+    "news": "Notícias",
+    "notification": "Notificação",
+    "notifications": "Notificações",
+    "play": "Jogar",
+    "player": "Jogador",
+    "players": "Jogadores",
+    "playing since :date": "Jogando desde :date",
+    "privacy": "Privacidade",
+    "privacy-policy": "Políticas de Privacidade",
+    "profile setting": "Configuração de Perfil",
+    "profile settings": "Configurações de Perfil",
+    "profile": "Perfil",
+    "profiles": "Perfis",
+    "recipient": "Recipiente",
```

```suggestion
    "recipient": "Destinatário",
```
Is this more accurate? This is in the context of messaging.

---

"Destinatário" is the correct one. "Recipiente" means "container" instead (it's a false friend).